I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 133, in script
    play music dictor8 volume 0.4
  File "renpy/common/000statements.rpy", line 123, in execute_play_music
    renpy.music.play(_audio_eval(p["file"]),
  File "renpy/common/000statements.rpy", line 32, in _audio_eval
    return eval(expr, locals=store.audio.__dict__)
  File "game/script.rpy", line 133, in <module>
    play music dictor8 volume 0.4
NameError: name 'dictor8' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 133, in script
    play music dictor8 volume 0.4
  File "C:\Users\Alexander\Downloads\renpy-8.0.3-sdk\renpy\ast.py", line 2232, in execute
    self.call("execute")
  File "C:\Users\Alexander\Downloads\renpy-8.0.3-sdk\renpy\ast.py", line 2220, in call
    return renpy.statements.call(method, parsed, *args, **kwargs)
  File "C:\Users\Alexander\Downloads\renpy-8.0.3-sdk\renpy\statements.py", line 281, in call
    return method(parsed, *args, **kwargs)
  File "renpy/common/000statements.rpy", line 123, in execute_play_music
    renpy.music.play(_audio_eval(p["file"]),
  File "renpy/common/000statements.rpy", line 32, in _audio_eval
    return eval(expr, locals=store.audio.__dict__)
  File "C:\Users\Alexander\Downloads\renpy-8.0.3-sdk\renpy\python.py", line 1092, in py_eval
    return py_eval_bytecode(code, globals, locals)
  File "C:\Users\Alexander\Downloads\renpy-8.0.3-sdk\renpy\python.py", line 1085, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/script.rpy", line 133, in <module>
    play music dictor8 volume 0.4
NameError: name 'dictor8' is not defined

Windows-10-10.0.19044 AMD64
Ren'Py 8.0.3.22090809
Rebirth 1.0
Fri Nov 11 23:53:02 2022
